src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js

```diff
@@ -142,41 +152,68 @@ function disableRow($containingForm, row) {
     $tdElements.parent().hide();
 }
 
-function updateEditTabAccordingToRecipient($containingForm) {
+function disableRowsAccordingToRecipient($containingForm) {
     var recipientType = $containingForm.find('select[name="recipienttype"]').val();
-    if (isRecipientsTeamMembersVisibilityOptionInvalidForRecipientType(recipientType)) {
-        // show the row Recipient(s) and hide the row Recipient's Team Members
-        enableRow($containingForm, ROW_RECIPIENT);
+    switch (recipientType) {
+    case 'SELF':
```

Comment:
It's actually because the reason for not showing the options are different, and I wanted to keep the separate comments for easy reference